graphs/DeltaArrow/index.js

```diff
@@ -9,22 +9,25 @@ module.exports = React.createClass({
 
   propTypes: {
     delta: React.PropTypes.number,
-    loading: React.PropTypes.bool
+    loading: React.PropTypes.bool,
+    emptyState: React.PropTypes.bool
   },
 
   getDefaultProps: function() {
     return {
-      loading: false
+      loading: false,
+      emptyState: false
     };
   },
 
   renderTriangle: function() {
-    var props   = this.props,
-        delta   = props.delta,
-        loading = props.loading,
+    var props      = this.props,
+        delta      = props.delta,
+        loading    = props.loading,
+        emptyState = props.emptyState,
         path;
 
-    if (delta == null || delta > 0 || loading == true) {
+    if (delta == null || delta > 0 || loading == true || emptyState == true) {
```

Comment:
Maybe move this into a var to make it a bit clearer. 
`var showBlankState = (delta === null || delta > 0 || loading === true || emptyState === true);`

Also you should use `===`. It's more equal. Look it up.